pagopa/io-web-profile
# IOPID-476 Responsive Adjustment

## Short description
Fixing responsive design based on design review.

## List of changes proposed in this pull request
- Fixing responsive design.
- Adding a snackbar for the copy code action.
- Adding the Play Store icon.

## How to test
Please check the following:
- Card's responsive behavior on the /profile page.
- Ensure that the background color is white on the /profileBlock page.
- Verify that the snackbar is triggered when attempting to access /profileBlockSuccess.
- Confirm that the Play Store icon has been changed on the logoutNoSession/l2/ page.
